app/src/main/java/org/lineageos/glimpse/viewmodels/MediaViewerViewModel.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * SPDX-FileCopyrightText: 2023 The LineageOS Project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.lineageos.glimpse.viewmodels
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProvider.AndroidViewModelFactory.Companion.APPLICATION_KEY
+import androidx.lifecycle.createSavedStateHandle
+import androidx.lifecycle.viewmodel.initializer
+import androidx.lifecycle.viewmodel.viewModelFactory
+import org.lineageos.glimpse.GlimpseApplication
+import org.lineageos.glimpse.repository.MediaRepository
+
+class MediaViewerViewModel(
```

Comment:
Why don't you add this stuff into the main one?

Reply:
Dunno if I already replied on Discord, but supposedly each fragment should have its own VM, we're fine with a single one since most of the functionalities are common, but this stuff has nothing to do with the other fragments